api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -393,6 +393,12 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	public ConceptSource getConceptSourceByUniqueId(String uniqueId);
 
 	/**
+	 * @see org.openmrs.api.ConceptService#getConceptSourceByHL7Code(java.lang.String)
+	 */
+	
```

Comment:
`      /* *
 	 * @verifies return null if given an empty string
 	 * @see ConceptService#getConceptSourceByHL7Code(String)
 	 */
 	@Test public void getConceptSourceByHL7Code_shouldReturnNullIfGivenAnEmptyString() throws Exception {
 
 		assertThat(conceptService.getConceptSourceByHL7Code(""), is(nullValue()));
 		assertThat(conceptService.getConceptSourceByHL7Code("    "), is(nullValue()));
 	}
 
 	/**
 	 * @verifies fail if given null
 	 * @see ConceptService#getConceptSourceByHL7Code(String)
 	 */`

Like this section in ConceptServiceTest.java